recipe/meta.yaml

```diff
@@ -29,21 +29,45 @@ requirements:
   host:
     - msinttypes  # [win and vc<14]
```

Comment:
It would probably be good to move these down to the first output

Reply:
I am not sure this would work in this model because 

- first conda executes the build.sh with the top level package & requirements, conda saves the host env
- then conda copies the requirements to the output _with the same name_ (xz)
- then conda executes the install.sh / install.bat script for the output (with the duplicated host env) and adds the same requirements to the output with the same name

This process is intransparent and a huge mess and something we totally want to do much cleaner with boa & the new recipe spec.